utility/import_exporter.h

```diff
@@ -0,0 +1,73 @@
+#ifndef IMPORT_EXPORTER_H
+#define IMPORT_EXPORTER_H
+
+#include "core/templates/map.h"
+#include "core/io/resource.h"
+#include "core/object/object.h"
+#include "core/os/file_access.h"
+#include "core/object/reference.h"
+#include "core/io/resource_importer.h"
+#include "resource_import_metadatav2.h"
+#include "import_info.h"
+
+class ImportExporter: public Reference {
+	GDCLASS(ImportExporter, Reference)
+	Array files;
+	String project_dir;
+	bool opt_bin2text = true;
+	bool opt_export_textures = true;
+	bool opt_export_samples = true;
+	bool opt_export_ogg = true;
+	bool opt_export_mp3 = true;
+	bool opt_lossy = true;
+	bool opt_rewrite_imd = true;
+	int ver_major = 0;
+	Vector<String> files_lossy_exported;
+	Vector<String> files_rewrote_metadata;
+	int import_count;
+	Vector<Ref<ImportInfo>> lossy_imports;
+	Vector<Ref<ImportInfo>> rewrote_metadata;
+	Vector<Ref<ImportInfo>> failed_rewrite_md;
+	Vector<Ref<ImportInfo>> failed;
+	Vector<Ref<ImportInfo>> success;
+	Vector<Ref<ImportInfo>> not_converted;
+
+	Error load_import_file(const String &p_path);
+	Error load_import_file_v2(const String &p_path);
+	Error rewrite_v2_import_metadata(const String &p_path, const String &p_dst, const String &p_res_name, const String &output_dir);
+	Error export_texture(const String &output_dir, Ref<ImportInfo> &iinfo);
+	Error export_sample(const String &output_dir, Ref<ImportInfo> &iinfo);
+	
+	Error _convert_tex_to_png(const String &output_dir, const String &p_path, const String &p_dst, String *r_name);
+	Error rewrite_import_data(const String &rel_dest_path, const String &output_dir, const Ref<ImportInfo> &iinfo);
+	Ref<ResourceImportMetadatav2> change_v2import_data(const String &p_path, const String &rel_dest_path, const String &p_res_name, const String &output_dir, const bool change_extension);
+
+	static Error ensure_dir(String &dst_dir);
```

Comment:
```suggestion
	static Error ensure_dir(const String &dst_dir);
```

It will cause multiple `non-const lvalue reference` errors otherwise.